server/server/server.go

```diff
@@ -233,13 +232,6 @@ func NewStore(config *config.Configuration, logger *mlog.Logger) (store.Store, e
 	if err != nil {
 		return nil, err
 	}
-	if config.AuthMode == MattermostAuthMod {
```

Comment:
Is this unrelated?

Reply:
This is intentional. `NewStore` isn't used in plugin mode. This previous code was doing nothing.

Reply:
Ok, intentional but unrelated.